eng/common/build.ps1

```diff
@@ -38,6 +38,9 @@ if($env:Platform) {
 function Print-Usage() {
   Write-Host "Common settings:"
   Write-Host "  -configuration <value>  Build configuration: 'Debug' or 'Release' (short: -c)"
+  Write-Host "                            'Debug', 'Release' (for .NET Core 2.1)"
+  Write-Host "                            'Debug-netcoreapp3_1', 'Release-netcoreapp3_1' (for .NET Core 3.1)"
```

Comment:
Files in eng/common are copied directly from Arcade. The next time we take an Arcade update, they are likely to be overwritten. So it is not a good idea to put these changes here. 

Can you please find a different mechanism?

Reply:
That is true. I have also explained the list of available configurations in our developer guide here. Would that not be enough, or should we always have the list of available configs printed out during `build -help`?

Reply:
In general, we should probably move away from the custom configurations and instead use "normal" multi-targeting, (i.e. `TargetFrameworks` in the .csproj).

We used custom configurations in the past because that's how build-tools required it to be. Now that we are using arcade, and a typical .NET build system, we should be able to only have `Debug` and `Release` configurations.

Reply:
We should always have a list of available configs during `build -help`.
But as @eerhardt pointed out below, let us switch to the more standard way of doing this. You can do that in a separate PR and leave these files as is.

---
In reply to: [538670694](https://github.com/dotnet/machinelearning/pull/5534#discussion_r538670694) [](ancestors = 538670694)

Reply:
Got it. Done.